extensions/amp-story/1.0/media-pool.js

```diff
@@ -878,6 +878,10 @@ export class MediaPool {
     if (mediaType == MediaType.VIDEO) {
       const ampVideoEl = domMediaEl.parentElement;
       if (ampVideoEl) {
+        const noaudio = ampVideoEl.hasAttribute('noaudio')
+        if(noaudio) {
+          domMediaEl.volume = 0
+        }
```

Comment:
I'd rather set the `volume` on `ampVideoEl` for consistency. It'll then follow the same logic as `<amp-video volume="0"`

```suggestion
        if (ampVideoEl.hasAttribute('noaudio')) {
          ampVideoEl.volume = 0;
        }
```